### PR TITLE
File Manager - Restore state after refresh

### PR DESCRIPTION
**Pull request by maxerokh**

### CODE REVIEW

File: js/ui/file_manager/file_items_controller.js [hunk 1]
```diff
@@ -177,6 +179,76 @@ export default class FileItemsController {
...
+            items: this._rootDirectoryInfo.items
+        };
+        const selectedKeyParts = this._getDirectoryPathKeyParts(this.getCurrentDirectory());
+
+        this._resetDirectoryState(this._rootDirectoryInfo);
+
+        this.setCurrentDirectory(null);
+        return this._refreshDeferred = this._loadItemsRecursive(this._rootDirectoryInfo, cachedRootInfo)
+            .then(() => {
+                var dirInfo = this._findSelectedDirectoryByPathKeyParts(selectedKeyParts);
```
Comment (vladkovl) [inline js/ui/file_manager/file_items_controller.js]: `var` => `const` is better

File: js/ui/file_manager/file_items_controller.js [hunk 2]
```diff
@@ -177,6 +179,76 @@ export default class FileItemsController {
...
+                this.setCurrentDirectory(dirInfo);
+
+                delete this._lockRefresh;
+            });
+    }
+
+    _loadItemsRecursive(directoryInfo, cachedDirectoryInfo) {
+        return this.getDirectories(directoryInfo)
+            .then(dirInfos => {
+                var itemDeferreds = [ ];
```
Comment (vladkovl) [inline js/ui/file_manager/file_items_controller.js]: `var` => `const` is better

Comment (maxerokh) [reply js/ui/file_manager/file_items_controller.js]: fixed

File: js/ui/file_manager/file_items_controller.js (hunk 1 above)
Comment (maxerokh) [reply js/ui/file_manager/file_items_controller.js]: fixed

**Review by vladkovl — APPROVED** [review]

**Pull request merged**